src/NUnitFramework/testdata/TestCaseSourceAttributeFixture.cs

```diff
@@ -142,6 +142,11 @@ public void MethodWithArrayArguments(object o)
         {
         }
 
+        [TestCaseSource(nameof(IncompatibleGenericTypeAndArgumentTestCases))]
+        public void MethodWithIncompatibleGenericTypeAndArgument(object o)
```

Comment:
This is not a generic method, and `NUnitTestCaseBuilder` doesn't even use the `TypeArgs`.